Add sizes and orientations for Template Starter (BL-4493)

andrew-polk opened the pull request.

andrew-polk commented:
Review status: 0 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/BloomExe/Book/SizeAndOrientation.cs, line 137 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636#-Kgo_l5QpnImXl4UNO4B:-Kgo_l5QpnImXl4UNO4C:bayh9jg) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/9263e1e22a01dd9ba48527341a17af49bbae4ab3/src/BloomExe/Book/SizeAndOrientation.cs#L137)):*
> ```Smalltalk
> 				if (fileName.ToLowerInvariant().Contains("mode") || fileName.ToLowerInvariant().Contains("page") ||
> 					fileName.ToLowerInvariant().Contains("matter") || fileName.ToLowerInvariant().Contains("languagedisplay") ||
> 					fileName.ToLowerInvariant().Contains("origami"))
> ```

This is not needed for this change. I saw the opportunity for a slight optimization while debugging. I'll split it out if you rather.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636)*
<!-- Sent from Reviewable.io -->

andrew-polk commented:
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/BloomBrowserUI/templates/template books/Template Starter/Template Starter.pug, line 14 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636#-KgogEfWH_mLRrt_q1vq:-Kgovt93IGhi2k1nSwuq:b-gx9f1) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/9263e1e22a01dd9ba48527341a17af49bbae4ab3/src/BloomBrowserUI/templates/template books/Template Starter/Template Starter.pug#L14)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

We should think about just using Basic Book.css. That's what Decodable and Leveled Reader templates use. Would that cause any problems?
</blockquote></details>

Assuming we want the same options as Basic Book, should be okay. I'll test it out.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636)*
<!-- Sent from Reviewable.io -->

andrew-polk commented:
Review status: 0 of 5 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/BloomBrowserUI/templates/template books/Template Starter/Template Starter.pug, line 14 at r1](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636#-KgogEfWH_mLRrt_q1vq:-Kgp2TWP__lBpEWeMPln:b-kgqwhq) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/9263e1e22a01dd9ba48527341a17af49bbae4ab3/src/BloomBrowserUI/templates/template books/Template Starter/Template Starter.pug#L14)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

Assuming we want the same options as Basic Book, should be okay. I'll test it out.
</blockquote></details>

I couldn't find any problems by testing

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636)*
<!-- Sent from Reviewable.io -->

hatton commented:
Review status: 0 of 5 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/BloomBrowserUI/templates/template books/Template Starter/TemplateStarter.less, line 1 at r2](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636#-KgyLqLY2QlVa-RATCkT:-KgyLqLY2QlVa-RATCkU:b-ti550p) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/de2bd7c4681668d43fbf49dd8a1a40ef6c706333/src/BloomBrowserUI/templates/template books/Template Starter/TemplateStarter.less#L1)):*
> ```Less
> /*The following is json. Bloom reads it in order to know what size/orientation options this stylesheet supports
> ```

Is this file still needed? Will the system pick these up from Basic Book.css?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636)*
<!-- Sent from Reviewable.io -->

andrew-polk commented:
Review status: 0 of 5 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/BloomBrowserUI/templates/template books/Template Starter/TemplateStarter.less, line 1 at r2](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636#-KgyLqLY2QlVa-RATCkT:-KgyS5rTiLwFMy1kaoK9:b-d723e4) ([raw file](https://github.com/bloombooks/bloomdesktop/blob/cbc601fee90016a0b27469f61587f1d3f2c2d3fe/src/BloomBrowserUI/templates/template books/Template Starter/TemplateStarter.less#L1)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

Is this file still needed? Will the system pick these up from Basic Book.css?
</blockquote></details>

Oops! I thought I had deleted it... Deleted now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomdesktop/1636)*
<!-- Sent from Reviewable.io -->
